docs/spark/spark-ddl.md

```diff
@@ -171,6 +171,14 @@ Iceberg uses table properties to control table behavior. For a list of available
 ALTER TABLE prod.db.sample UNSET TBLPROPERTIES ('read.split.target-size')
 ```
 
+`SET TABLE COMMENT` Using SET TBLPROPERTIES
```

Comment:
This line gives an impression that `SET TABLE COMMENT` is a SQL command, which is not the case.